Makefile

```diff
@@ -474,8 +474,23 @@ drone:
 
 # support go modules
 check-mod:
+ifeq ($(BUILD_IN_CONTAINER),true)
+	docker run \
+		--entrypoint "" \
+		-e "GO111MODULE=on" \
+		-e "GOPROXY=https://proxy.golang.org" \
+		-v ~/go/pkg/mod:/go/pkg/mod \
+		-v $(shell pwd):/src/loki \
+		$(IMAGE_PREFIX)/loki-build-image:$(BUILD_IMAGE_VERSION) \
+		bash -c "cd /src/loki; \
+		go mod download; \
+		go mod verify; \
+		go mod tidy; \
+		go mod vendor; "
```

Comment:
Is it possible for this to re-run the make command like we do in other instances of BUILD_IN_CONTAINER?

Reply:
Good catch.  Latest push should resolve.